application/src/test/java/org/togetherjava/tjbot/jda/JdaTester.java

```diff
@@ -354,6 +367,19 @@ public TextChannel getTextChannelSpy() {
         return textChannel;
     }
 
+    /**
+     * Gets the text channel spy used as universal text channel by all mocks created by this tester
+     * instance.
+     * <p>
+     * For example the events created by {@link #createSlashCommandInteractionEvent(SlashCommand)}
+     * will return this spy on several of their methods.
+     *
+     * @return the text channel spy used by this tester
+     */
```

Comment:
copy pasta? u seem to have forgotten to replace "text" by "thread" :)

Reply:
Oops

Reply:
Changed